{
  "id": "CVE-2024-45230",
  "published": "2024-10-08T16:15:11",
  "modified": "2025-09-24T12:29:34.525353",
  "details": "An issue was discovered in Django 5.1 before 5.1.1, 5.0 before 5.0.9, and 4.2 before 4.2.16. The urlize() and urlizetrunc() template filters are subject to a potential denial-of-service attack via very large inputs with a specific sequence of characters.",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
    }
  ],
  "repo_url": "https://github.com/django/django",
  "cwes": [],
  "commits": [
    "2591fb8d4c0246f68b79554976c012039df75359",
    "3bf46e2e023904c825728267f3131c42f02ce696"
  ]
}
